app/xlist/scripts/HomeController.js

```diff
@@ -86,13 +86,15 @@ angular
 
     var THRESHOLD = 0.25;
 
-    var findNear = function(location) {
+    var findNearAndPassLocation = function(location) {
       _.each($scope.pairs, function(pair) {
         var distance = getDistance(location, pair.geoList.location);
         if (distance < THRESHOLD) {
           pushNear(pair);
         }
       });
+      // used to pass location to location controller
+      supersonic.data.channel('location').publish(location);
```

Comment:
Actually I take this back, it might make sense to update the bounds of the search every 10 seconds, but it might be unnecessary. I thought we could just query the device location once when loading the LocationController and use that as the bounds.